.node-version

```diff
@@ -1 +1 @@
-8.2.1
+10.11.0
```

Comment:
10.15.1 is the latest LTS

Reply:
@chrisbreiding: Why did you marked this as resolved?

Reply:
We don't want to upgrade to the latest Node, but rather the version that Electron includes, since almost everything in this repo is packaged up and run in Electron.